src/steps/powershell.rs

```diff
@@ -67,21 +67,34 @@ impl Powershell {
 
         print_separator(t!("Powershell Modules Update"));
 
-        let mut cmd = vec!["Update-Module"];
+        let unload_cmd = ["Get-Module | Remove-Module -Force"];
+        let mut update_cmd = vec!["Update-Module"];
+        let reload_cmd = ["Get-Module -ListAvailable | Import-Module"];
 
         if ctx.config().verbose() {
-            cmd.push("-Verbose")
+            update_cmd.push("-Verbose");
         }
 
         if ctx.config().yes(Step::Powershell) {
-            cmd.push("-Force")
+            update_cmd.push("-Force");
         }
 
+        println!("{}", t!("Unloading modules..."));
```

Comment:
Topgrade has added support for i18n, so we need to [add translations](https://github.com/topgrade-rs/topgrade/blob/444689c899bc6a26c696fcfbccd0ef574a8f7870/locales/app.yml#L63-L67) for this "Unloading modules..." and "Reloading modules..." in line 94.

You can use ChatGPT or other LLM to get them translated.